src/toggleTask.js

```diff
@@ -0,0 +1,35 @@
+import './mainContainer.css'
+import previousBtn from './image/left-arrow.svg'
+import nextBtn from './image/right-arrow.svg'
+import addTasksBtn from './image/add-tasks.svg'
+import TaskCard from './taskCard';
+
+const ToggleTaskDays = () =>{
+return(
+    <div className='toggle'>
+      <a className='previousTask'> <img src={previousBtn} alt="previous button" width="80" height="80" /></a>
```

Comment:
```suggestion
      <a className='previousTask' href="#"> <img src={previousBtn} alt="previous button" width="80" height="80" /></a>
```
Always use `href` attribute while using `<a>` anchor tag